ArduCopter/commands.cpp

```diff
@@ -51,6 +51,9 @@ bool Copter::set_home_to_current_location() {
 bool Copter::set_home_to_current_location_and_lock()
 {
     if (set_home_to_current_location()) {
+        // tell barometer about GPS altitude (for air pressure drift compensation)
+        barometer.set_home_alt(float(gps.location().alt)/100.f);
```

Comment:
no need to cast a float to a float especially if the function is already expecting a float. Also, please follow the other examples of casting where (float)x is used instead of float(x)